changelogs/fragments/fix_nxos_smu.yaml

```diff
@@ -0,0 +1,3 @@
+---
```

Comment:
Could you also please add the following?
```yaml
deprecated_features:
  - Deprecated `nxos_smu` in favor of `nxos_rpm` module.
```

Reply:
Done